assets/js/blocks/cart-checkout/cart/full-cart/style.scss

```diff
@@ -98,45 +98,34 @@ table.wc-block-cart-items {
 	h2 span {
 		min-width: 33%;
 	}
-	.wc-block-cart-item__price,
-	.wc-block-cart-item__individual-price,
-	.wc-block-cart-item__product-metadata,
-	.wc-block-cart-item__image > *,
+	.wc-block-components-product-price,
+	.wc-block-components-product-metadata,
```

Comment:
For some reason, some of the class names used in the Cart skeleton didn't match the real React components. I tried to unify them here.